### ISSUE TITLE
Multi-GPU can not run

### ISSUE THREAD

**Issue by yangpanquan** [ZhanYang-nwpu/Mono3DVG#13]
Hi, author, thanks for your amazing work. I encounter some problem while debugging your code.
when I run the code with one RTX3090, there is no problem.Because the training time is too long with one gpu, so, I try to run with two RTX 3090,  there is something wrong.I just modify the parameter "gpu_ids" with "0, 1" .Please see the picture below.I will appreciate it if you solve it.

![image](https://github.com/user-attachments/assets/cf2b4f69-16d2-4749-a37b-ca19527bbe6d)
![image](https://github.com/user-attachments/assets/40971ca8-e654-42b4-aaaf-854e3dc32abb)
![image](https://github.com/user-attachments/assets/3dc58ad2-9750-4fb8-97c7-88637e00254a)

**Comment by xf-zh** [comment]
您好，请问您这个问题解决了吗？

**Comment by WeixuanQiao** [comment]
I also encountered this problem. Is there a solution？

**Comment by WeixuanQiao** [comment]
I checked the input type of the model and found that 'text' and 'im_name' are 'list' rather than 'tensor', so they cannot be automatically distributed by DataParallel to different GPUs.

this is a naive solution.
add following code in mono3dvg.py's forward function of Mono3DVG ( in line 142):
```
text = text[torch.cuda.current_device():len(text):torch.cuda.device_count()]
im_name = im_name[torch.cuda.current_device():len(text):torch.cuda.device_count()]
```
and modify property 'shuffle' of train_loader(in dataloader_helper.py) to False.
